src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -161,8 +159,25 @@ const PastAnalysisTable = () => {
       }
       setLoading(false);
     })();
-  }, [tableStates.maxFeedId, tableStates.page, perpage, dispatch, history, stagingDcmImages]);
-
+  }, [tableStates, perpage, stagingDcmImages, dispatch]);
+
+
+  // Polls ChRIS backend and refreshes table if any of the plugins with the given IDs have a terminated status
+  useInterval(async () => {
+    if (tableStates.processingPluginIds) {
+      for (const id of tableStates.processingPluginIds) {
+        const refresh: boolean = await ChrisIntegration.checkIfPluginTerminated(id);
```

Comment:
It's okay to leave it as it is but just an FYI since `checkIfPluginTerminated()` resolves to a `boolean`, `refresh` implicitly gets the `boolean` type.

Reply:
Why is it not necessary in this case to specify a type when it is known?